testng/src/main/java/io/cucumber/testng/TestNGCucumberRunner.java

```diff
@@ -73,9 +85,13 @@ public TestNGCucumberRunner(Class<?> clazz) {
                 .parse(clazz)
                 .build(propertiesFileOptions);
 
+        RuntimeOptions otherOptions = new CucumberPropertiesParser()
+                .parse(properties::get)
```

Comment:
Is this correct ? shows an error in eclipse for me

Reply:
Yes. `parse` accepts a `CucumberPropertiesProvider` which is a functional interface with a signature of `String -> String`. 
`properties::get` is also function of the form `String -> String`.  Maybe you have to annotate both `CucumberPropertiesProvider` with `@FunctionalInterface` for eclipse to understand.

Reply:
Error was gone after importing core to workspace